biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -139,37 +206,30 @@ export const SimulatePage = () => {
           <div className="simboard-container">
             <h2>Simulation Board</h2>
             <div className="simboard-controls">
-              <button onClick={() => playSimulation()}>Play</button>
-              <button onClick={() => setCurrentStateFrame(0)}>Reset</button>
-            </div>
-            <div>
-              Time of day:{' '}
-              {currentStateFrame % 12 === 0 ? 12 : currentStateFrame % 12}:00{' '}
-              {currentStateFrame >= 12 ? 'PM' : 'AM'}
+              <button
+                onClick={() => {
+                  const pauseFn = play();
+                  setControlSimultation({ pauseFN: pauseFn });
```

Comment:
If I click play on a simulation and let it run for a bit, then (without hitting anything else), I click play again, I end up having two simulations running at the same time. I would recommend one of two options:
(1) one button, two states: when a user clicks "play", the button changes to say "pause". If a user clicks on it now, the simulation will pause, and the button text will again read "play" 
(2) before a user clicks "play", the pause button is disabled. When a user clicks "play", the pause button becomes enabled, but the "play" button is disabled. "Play" only becomes enabled again when "pause" is clicked

Reply:
Fixed! (I hope :))